tests/app/position_PositionShare_Lock_Event_GET_test.py

```diff
@@ -153,7 +237,15 @@ def setup_data(self, session: Session, token_address: str, base_time: datetime):
 
     # Normal_1
     # List all Events
-    def test_normal_1(self, client: TestClient, session: Session):
+    @pytest.mark.parametrize(
+        "get_params",
+        [
+            {"include_token_details": True},
+            {"include_token_details": False},
+            {},
+        ],
+    )
```

Comment:
Made test cases parametrize to ensure both of response data(token_detail is include and not) is valid.